client/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@date-io/date-fns": "^2.11.0",
+    "@date-io/date-fns": "^1.3.13",
```

Comment:
Did we downgrade this package on purpose?

Reply:
Yep! There's an issue with higher versions of date-fns. I installed it in a previous ticket as it was missing after the bookings UI was merged to develop. Bug report here recommended downgrading to that package version specifically.  https://github.com/mui-org/material-ui-pickers/issues/1440